tester.go

```diff
@@ -98,8 +98,8 @@ func preProcessTest(test *Test, defaultHost string) error {
 
 	// Method
 	method := stringValue(test.Request.Method, "GET")
-	if method != "GET" && method != "POST" && method != "PUT" && method != "PATCH" && method != "DELETE" && method != "HEAD" && method != "OPTIONS" {
-		return fmt.Errorf("invalid method %s. only GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS are supported", method)
+	if method != "GET" && method != "POST" && method != "PUT" && method != "PATCH" && method != "DELETE" && method != "HEAD" && method != "OPTIONS" && method != "PURGE" && method != "PROPFIND" {
```

Comment:
Probably being super finical here, but could this be changed to a function call that validates methods like 
```
func isValidMethodType {
 Check for valid method return boolean....
}
```